src/ontology/cl-edit.owl

```diff
@@ -5623,12 +5624,10 @@ EquivalentClasses(obo:CL_0000227 ObjectIntersectionOf(obo:CL_0000000 ObjectSomeV
 
 # Class: obo:CL_0000228 (multinucleate cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FB:ma") obo:IAO_0000115 obo:CL_0000228 "A cell with more than one nucleus.")
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000228 "AEO:0000203")
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000228 "WBbt:0008074")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000228 "syncitium")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000228 "syncytial cell")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000228 "syncytium")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FB:ma") Annotation(oboInOwl:hasDbXref "Wikipedia:Multinucleate") obo:IAO_0000115 obo:CL_0000228 "A cell with more than one nucleus.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "Wikipedia:Multinucleate") oboInOwl:hasExactSynonym obo:CL_0000228 "multinucleated cells")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "Wikipedia:Multinucleate") oboInOwl:hasExactSynonym obo:CL_0000228 "polynuclear cells")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WBbt:0008074") Annotation(oboInOwl:hasDbXref "Wikipedia:Syncytium") oboInOwl:hasRelatedSynonym obo:CL_0000228 "syncytium")
```

Comment:
I would think it is a narrow synonym, since it is a subclass. However, I would remove it.